src/pages/[lang]/work-in-progress.tsx

```diff
@@ -19,7 +19,7 @@ const IndexPage: NextPage = () => (
       <meta name="viewport" content="initial-scale=1.0, width=device-width" />
     </Head>
     <Hero />
-    <EventDetails />
+    <EventDetailsExtended />
```

Comment:
copy-pasting and extending the section accordingly to leave https://code.akai.org.pl/ intact while at the same time adding some more info for https://code.akai.org.pl/en/work-in-progress 😉